### DISCUSSION TITLE
Manual installation of dashboard gives blank screen

### DISCUSSION THREAD

**Discussion by MaryAS1996** [mirumee/saleor-dashboard#3768] [Q&A]
After cloning from the git and installing all the dependencies the starting of the localhost:9000 only gives a blank screen. what could be the reason?

**Comment by kzuraw** [comment]
Hi, @MaryAS1996 thanks for opening the discussion 🎉. Can you give more details:
* What OS and browser do you use?
* Are there errors inside the browser/terminal?
* Do you clone the main branch or maybe another branch?

**Reply by codal-nikhilbhalodi** [reply]
I also facing the same issue. 
- OS `Ubuntu 20.04.06 LTS`
- In the terminal, it's showing nothing and in the browser, it keeps loading. In the network console, it shows nothing.
- I'm using the 3.14 release code.

**Reply by erioernesto02** [reply]
In my case I also get a blank screen and in the firefox developer tools I get status 304.

1. OS Windows 11 (64)
2. Using the 3.14.5 version.

**Reply by codal-nikhilbhalodi** [reply]
I find the solution for my issue in the Chrome browser. Just follow these `Chrome settings -> System -> Use hardware acceleration when available ->  Turn Off`
After that reload the browser and work for me.
